api_app/tests/test_serializers.py

```diff
@@ -87,3 +90,126 @@ def test_user_serialization_missing_data(self):
         self.assertIsNone(serialized_data.validated_data.get('first_name'))
         self.assertIsNone(serialized_data.validated_data.get('last_name'))
         self.assertIsNone(serialized_data.validated_data.get('email'))
+
+
+class ProductSerializerTests(TestCase):
+
+    def setUp(self):
+        self.fake = Faker()
+        self.product_data = ProductFactory()
+        self.serializer_input_data = {
+            'name': self.fake.text(max_nb_chars=255),
+            'serving_size': self.fake.random_int(min=1, max=5000),
+            'calories': self.fake.random_int(min=1, max=10000),
+            'protein': self.fake.pyfloat(left_digits=2, right_digits=2, positive=True),
+            'carbohydrate': self.fake.pyfloat(left_digits=2, right_digits=2, positive=True),
+            'fat': self.fake.pyfloat(left_digits=2, right_digits=2, positive=True),
+        }
+
+        self.serializer = ProductSerializer(instance=self.product_data)
+
+    def tearDown(self):
+
+        if hasattr(self, 'created_instance'):
+            self.created_instance.delete()
+
+        super().tearDown()
+
+    def test_product_serialization(self):
+        serialized_data = self.serializer.data
+
+        self.assertEqual(serialized_data['name'], self.product_data.name)
+        self.assertEqual(serialized_data['serving_size'], self.product_data.serving_size)
```

Comment:
zerknij ile kodu powtarzasz
6x self.assertEqual(serialized_data['serving_size'], self.product_data.serving_size)
spróbuj napisać to tak aby takie sprawdzenie było tylko raz